ilastik/applets/objectExtraction/opObjectExtraction.py

```diff
@@ -576,7 +588,27 @@ def _augmentFeatureNames(self, features):
 
         return feature_names_with_default
 
-    def _extract(self, image, labels):
+    def _createAtlasMapping(self, region_centers:numpy.ndarray, atlasImage:numpy.ndarray):
+        axes = {tag.key:idx for idx, tag in  enumerate(atlasImage.axistags)}
+        num_center_points = len(region_centers)
+        num_channels_in_atlas = atlasImage.shape[axes['c']] if 'c' in axes else 1
+        atlas_mapping = numpy.zeros(num_center_points * num_channels_in_atlas)
+        atlas_mapping = atlas_mapping.reshape(num_center_points, num_channels_in_atlas)
```

Comment:
```suggestion

```

Reply:
So  a few lines below there is this: `atlas_mapping[obj_idx] = atlas_value`; `atlas_value` is simply a sampling of the Atlas image, so it might come back as a multi-channel point if the user is using a multi-channel image. That's why I have to reshape the atlas_mapping, so that every item in it has as many channels as the sampling from the atlas